test/hotspot/jtreg/runtime/cds/appcds/FillerObjectLoadTest.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2022 THL A29 Limited, a Tencent company. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/**
+ * @test
+ * bug 8286066
+ * @summary VM crash caused by unloaded FillerObject_klass
+ * @library /test/lib
+ * @requires vm.cds
+ * @run driver FillerObjectLoadTest
+ */
+
+import jdk.test.lib.process.OutputAnalyzer;
+import jdk.test.lib.process.ProcessTools;
+
+public class FillerObjectLoadTest {
+    public static void main(String... args) throws Exception {
+        ProcessBuilder pb = ProcessTools.createJavaProcessBuilder(
+                "-XX:+IgnoreUnrecognizedVMOptions", "-XX:-UseCompressedClassPointers",
+                "-XX:+UnlockExperimentalVMOptions", "-XX:+UseEpsilonGC", "-Xshare:dump",
+                "-XX:SharedArchiveFile=./hello.jsa");
```

Comment:
Use ` TestCommon.getCurrentArchiveName()` to get a unique name to use.

Reply:
Actually that might be overkill for this simple case ... I wonder if @iklam  could comment?

hello.jsa is a bit of a weird name to use though :)

Reply:
> Use ` TestCommon.getCurrentArchiveName()` to get a unique name to use.

Hi @dholmes-ora , I didn't get the point why we need to get a unique name.
What's the problem if we always use the same name?
Thanks.

Reply:
We had issues in the past with tests running concurrently IIRC . I'm not sure if this is still an issue if the test working directory is already unique. @iklam should remember the details. We put in a lot of support code to make sure things always worked smoothly.

Reply:
We had problems on Windows in the past. If two tests generate an archive with the same name, after the first test finishes, jtreg deletes the first version of this file. However, Windows virus scanners will keep the file alive for a little longer. When the second tests tries to create the file again, it gets a file permission error. This would lead to infrequent random failures in the CDS tests.

That's the reason we use `TestCommon.getCurrentArchiveName()` in the CDS tests.

Reply:
> We had issues in the past with tests running concurrently IIRC . I'm not sure if this is still an issue if the test working directory is already unique. 

The archive file is generated in the test's "working scratch" directory, which is reused. For example, if you have 3 test cases and a jtreg concurrency setting of 2:  `0/scratch/` could be used twice whereas `1/scratch/` is used once.

Reply:
> We had problems on Windows in the past. If two tests generate an archive with the same name, after the first test finishes, jtreg deletes the first version of this file. However, Windows virus scanners will keep the file alive for a little longer. When the second tests tries to create the file again, it gets a file permission error. This would lead to infrequent random failures in the CDS tests.
> 
> That's the reason we use `TestCommon.getCurrentArchiveName()` in the CDS tests.

Thanks @iklam and @dholmes-ora for your kind explanation.

Updated.
`TestCommon.getCurrentArchiveName()` returns `null` so I use `TestCommon.getNewArchiveName()` instead.
Thanks.